sdk/core/core-amqp/src/util/utils.ts

```diff
@@ -175,48 +174,21 @@ export class Timeout {
  * @param abortErrorMsg - The abort error message associated with containing operation.
  * @param value - The value to be resolved with after a timeout of t milliseconds.
  * @returns - Resolved promise
+ * @deprecated
```

Comment:
Could we add a comment to the `@deprecated` tag to recommend using `delay` from `@azure/core-util`?

Reply:
Yeah can you add a comment saying `This function is deprecated please use delay from @azure/core-util`

Reply:
Done